Contents/Code/plex_api_helper.py

```diff
@@ -298,6 +310,9 @@ def add_media(item, media_type, media_url_id, media_file=None, media_url=None):
         new_themerr_data[media_type_dict[media_type]['themerr_data_key']] = media_url_id
 
         general_helper.update_themerr_data_file(item=item, new_themerr_data=new_themerr_data)
+
+        # unlock the field since it contains an automatically added value
+        getattr(item, "_edit")(**{'{}.locked'.format(media_type_dict[media_type]['plex_field']): 0})
```

Comment:
```suggestion
        edit_field = "{}.locked".format(media_type_dict[media_type]['plex_field'])
        edits = {
            edit_field : 0,
        }
        item.edit(**edits)
```

Would `item.edit` work? https://github.com/pkkid/python-plexapi/blob/46c20a598f78c88fad3df1240ad4b53f356ab894/plexapi/base.py#L588

Also, just made this a bit more readable.

Reply:
This works on my local setup